File: backend/core/config/ImageConfig.py
import os, logging

from core.utils.state.DirectoryState import DirectoryState

class ImageConfig:

    @staticmethod
    def initImageDirectory():
        try:
            os.makedirs(DirectoryState.IMAGE_DIR, exist_ok=True)
            logging.info('✅ 图片文件夹初始化成功')
        except Exception as e:
            logging.error(f'❌ 初始化图片文件夹失败: { str(e) }')

File: backend/src/core/config/ImageConfig.ts
import fs from 'fs';
import { DirectoryState } from '../utils/state/DirectoryState';
import { LogConfig } from './LogConfig';

export async function initilizeImageDirectory(): Promise<void> {
    try {
        fs.mkdirSync(DirectoryState.IMAGE_DIR, { recursive: true })
        LogConfig.initLogging().info('✅ 图片文件夹初始化成功')
    } catch (error) {
        LogConfig.initLogging().error('❌ 图片文件夹初始化失败', error)
        throw error
    }
}